lms/djangoapps/course_api/views.py

```diff
@@ -321,13 +326,13 @@ def get_queryset(self):
         form = CourseListGetForm(self.request.query_params, initial={'requesting_user': self.request.user})
         if not form.is_valid():
             raise ValidationError(form.errors)
-
```

Comment:
Nit: was removing this line intentional?

Reply:
Yes. And Sorry. I've come to create this "habit" of removing all blank lines in methods if there is no space between the first statement and the docstring. Do you want me to put back the line?